app/scripts/proactive/config.js

```diff
@@ -95,17 +95,17 @@ define(function () {
         },
         languages_content_type: {
             "scalaw" : "text/x-scala",
-            "groovy" : "application/x-groovy",
-            "javascript" : "application/javascript",
-            "python" : "application/x-python",
-            "cpython" : "application/x-python",
-            "ruby" : "application/x-ruby",
+            "groovy" : "text/x-groovy",
+            "javascript" : "text/javascript",
```

Comment:
@chloegugli Why are they becoming text/xxxxx now? (I know it is a common debate on whether this is meant to be manipulated by machine or read by humans)

Reply:
Actually this is a fix for "-Groovy scripts (and others) were displayed as text in code mirror of catalog portal". So I had a look at which content-types code mirror understands : https://codemirror.net/mode/

Reply:
👍